### PR TITLE
Fix not working github action to update translation with coverage deltas

### PR DESCRIPTION
**Pull request by st-Wook**
1. Fixed an issue where the action was not working due to conditionality.
2. Fixed an error that occurred when attempting to fetch file changes during local testing.
3. Changed to a more readable comment format.
4. Modified the action to update existing comments continuously rather than adding new ones.

[ Comment format ]
image

### CODE REVIEW

File: .github/workflows/pull-requests.yml [hunk 1]
```diff
@@ -1,29 +1,48 @@
...
+          echo "" >> sumup.md
+          echo "| Language | Change |" >> sumup.md
+          echo "| --- | --- |" >> sumup.md
           for index in ${!ADDS[@]}; do
-            echo ${LANS[index]} | sed -E "s/_/ /g" >> sumup.txt
-            DELTA=$(bc <<< "scale=2;(${ADDS[index]}-${SUBS[index]})*100/1")
-            echo $DELTA % >> sumup.txt
+            LANGUAGE=${LANS[index]//_/ }
+            DELTA=$(echo "scale=6; (${ADDS[index]} - ${SUBS[index]})*100" | bc)
+            echo "| $LANGUAGE | $(printf "%.6f" $DELTA) % |" >> sumup.md
```
Comment (AlexandreSi) [inline .github/workflows/pull-requests.yml:32]: You can use only 2 or 3 decimals I think. I had made this action in order to help for the check me make before merging the PR: Make sure there isn't a massive drop in translation ratios (that would mean there has been a problem somewhere on crowdin).

**Review by AlexandreSi — APPROVED** [review]
Great work!
Thanks for improving this action I had more or less abandoned

File: .github/workflows/pull-requests.yml (hunk 1 above)
Comment (st-Wook) [reply .github/workflows/pull-requests.yml:32]: Ahh, I see... How about we avoid commenting on the items that are showing as 0%?
image

Comment (4ian) [reply .github/workflows/pull-requests.yml:32]: That's a good idea if it's not too complex. Otherwise, it's already very nice!

**Review by 4ian — APPROVED** [review]
Looks really cool, thank you!

File: .github/workflows/pull-requests.yml (hunk 1 above)
Comment (st-Wook) [reply .github/workflows/pull-requests.yml]: Done! You can see the appearance of the table after changes in the screenshot in the PR description!

**Pull request merged**